packages/next/src/client/components/react-dev-overlay/internal/container/RuntimeError/index.tsx

```diff
@@ -67,7 +68,6 @@ export function RuntimeError({ error }: RuntimeErrorProps) {
           <h2>Source</h2>
           <GroupedStackFrames
             groupedStackFrames={leadingFramesGroupedByFramework}
-            show={all}
```

Comment:
This might hide the call stacks, but we don't need it since we already group the stacks into react and nextjs group